SharedContent/README.md

```diff
@@ -0,0 +1,9 @@
+#  Windows App SDK Sample Templates
+
+The `SharedContents` folder contains sample templates for C# and C++ (using WinUI 3). To use the current templates, sample authors should follow these steps:
+
+1. Clone the Windows App SDK samples repo and then copy all the files under the appropriate template folder to your feature sample directory. For example, create a new folder for your sample called `Samples/MyFeatureName/cs-winui/`, and copy all the contents under `SharedContent/cs-winui-template` to this new folder.
```

Comment:
It would be good to have a VSIX eventually, but we will need to figure out how to update these when new versions of WinAppSDK are released. Is there a way to generate a unique identity without a VSIX?

Reply:
We will work on creating a VSIX for the samples